src/main/java/frc/robot/RobotContainer.java

```diff
@@ -141,7 +147,14 @@ public boolean getAsBoolean() {
   DoubleSupplier m_translationXSupplier = new DoubleSupplier() {
     @Override
     public double getAsDouble() {
-      var input = -modifyAxis(joystick0.getY()) * Constants.TRAINING_WHEELS;
+      var input = 0.0;
+      var robotState = drivetrainSubsystem.getGyroscopeRotation();
+      if (intakeDown && intakeSubsystem != null) {
+        input = -modifyAxis(joystick0.getY()) - modifyAxis((robotState.getSin() * joystick4.getY() + robotState.getCos() * joystick4.getX()) * Constants.OWEN_WHEELZ) * Constants.TRAINING_WHEELS;
```

Comment:
The operator joystick should adjust the input, that combined input should be passed through modifyAxis() 

- [ ] compute combined input and pass through a single modifyAxis()